src/ashell/ide-comms.h

```diff
@@ -0,0 +1,67 @@
+// Copyright (c) 2017-2018, Intel Corporation.
+
+#ifndef __ashell_comms_h__
+#define __ashell_comms_h__
+
+#include <arch/cpu.h>
```

Comment:
split into to C includes and Zephyr includes

Reply:
Done.